src/components/OpenSeadragonViewer.js

```diff
@@ -85,7 +85,12 @@ export class OpenSeadragonViewer extends Component {
         viewerConfig.flip !== undefined && viewer.viewport.setFlip(viewerConfig.flip);
       }
 
-      this.addAllImageSources(!(viewerConfig));
+      const canvasesChanged = !(isEqual(canvasWorld.canvasIds, prevProps.canvasWorld.canvasIds));
```

Comment:
`canvasesChanged` is already defined here:
https://github.com/ProjectMirador/mirador/blob/851b4a5f29c2dd8015283978e232bb6f48d9726c/src/components/OpenSeadragonViewer.js#L97

Is this redundant?

Reply:
Yeah I double checked. This was added in a spot it already was. Removed it.